demo/site/src/util/recursivelyLoadBlockData.ts

```diff
@@ -0,0 +1,11 @@
+import { BlockLoader, BlockLoaderDependencies, recursivelyLoadBlockData as cometRecursivelyLoadBlockData } from "@comet/cms-site";
+
+const blockLoaders: Record<string, BlockLoader> = {
+    // Add your block loaders here
```

Comment:
Please don't remove the block loaders and module augmentation.

Reply:
1d8c1e0fd80e83c125ddc9d49810da5d58b2749b